### PR TITLE
Deprecating "cordova-plugin-livereload", since "taco run --livereload" is the supported way of using it.

### PR DESCRIPTION
**Pull request by nisheetjain**
screen shot 2016-01-25 at 7 39 06 pm

### CODE REVIEW

**Comment by nisheetjain** [comment]
@omefire , @subhagpo, @nikhilkh can you take a look

**Comment by omefire** [comment]
As part of this PR, we should also update the README.md to point to TACO.

**Comment by nisheetjain** [comment]
added Deprecation notice to README.md

File: deprecatePluginHook.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/**
+ *******************************************************
+ *                                                     *
+ *   Copyright (C) Microsoft. All rights reserved.     *
+ *                                                     *
+ *******************************************************
+ */
+console.error("\x1b[31m");
```
Comment (omefire) [inline deprecatePluginHook.js:8]: what's this for ? Maybe add a comment ?

**Comment by omefire** [comment]
LGTM !

File: deprecatePluginHook.js (hunk 1 above)
Comment (nisheetjain) [inline deprecatePluginHook.js:8]: to print it in red color.

**Pull request merged**